project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -235,3 +229,36 @@ function c4m_user_notifications_notify_inactive_users() {
  */
 function c4m_user_notifications_notify_comment_on_content($comment) {
 }
+
+/**
+ * Email notification to group requester, that group is waiting for review.
+ */
+function c4m_user_notifications_notify_new_group_pending($node) {
```

Comment:
We require the empty function, because it's used in this pr as trigger:
https://github.com/capacity4dev/capacity4more/pull/1059/files/d284a2a355cceafa44062dca9315912e11d48d88#diff-751d9add9b894092db2bb18dbee6622fR373